### PR TITLE
GUI: Display game title at the top of the GMM

### PR DESCRIPTION
**Pull request by ashutosh97**

### CODE REVIEW

File: base/main.cpp [hunk 1]
```diff
@@ -209,12 +208,14 @@ static Common::Error runGame(const EnginePlugin *plugin, OSystem &system, const
 	Common::String caption(ConfMan.get("description"));
 
 	if (caption.empty()) {
-		caption = EngineMan.findGame(ConfMan.get("gameid")).description();
+	    caption = EngineMan.findGame(ConfMan.get("gameid")).description();
```
Comment (criezy) [inline base/main.cpp]: We use tabs and not spaces for indentation.

Actually it doesn't look like any of the changes in this file do anything, so I would suggest to just drop all these changes from your commit.

File: engines/dialogs.cpp [hunk 2]
```diff
@@ -20,13 +20,25 @@
  *
  */
 
+
+#include "base/version.h"
```
Comment (criezy) [inline engines/dialogs.cpp]: This header file is already included a few lines below.

File: engines/dialogs.cpp [hunk 3]
```diff
@@ -47,6 +60,9 @@
 #include "gui/KeysDialog.h"
 #endif
 
+
+		
+
```
Comment (criezy) [inline engines/dialogs.cpp]: Why did you add those three blank lines?

File: engines/dialogs.cpp [hunk 4]
```diff
@@ -78,9 +102,10 @@ MainMenuDialog::MainMenuDialog(Engine *engine)
 	GUI::StaticTextWidget *title = new GUI::StaticTextWidget(this, "GlobalMenu.Title", "ScummVM");
 	title->setAlign(Graphics::kTextAlignCenter);
 #endif
-
-	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", gScummVMVersionDate);
+     
+	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", caption.c_str()) ;
```
Comment (criezy) [inline engines/dialogs.cpp]: The description of the task we have on our wiki is "In addition to the ScummVM logo and version, how about showing the engine name and game title at the top of the GMM?" It doesn't say anything about removing the version.

Can you explain why you took the decision to remove the version? Is there is good reason for doing it?

File: base/main.cpp (hunk 1 above)
Comment (ashutosh97) [inline base/main.cpp]: Done.

File: engines/dialogs.cpp (hunk 2 above)
Comment (ashutosh97) [inline engines/dialogs.cpp]: Removed the duplicate one.

File: engines/dialogs.cpp [hunk 5]
```diff
@@ -20,13 +20,25 @@
...
 #include "common/system.h"
 #include "common/translation.h"
+#include "common/archive.h"
+#include "common/debug.h"
+#include "common/debug-channels.h" 
+#include "gui/EventRecorder.h"
+#include "common/fs.h"
+#include "common/textconsole.h"
+#include "common/tokenizer.h"
+#include "common/osd_message_queue.h"
```
Comment (ashutosh97) [inline engines/dialogs.cpp]: Removed all the unnecessary header files.

File: engines/dialogs.cpp (hunk 3 above)
Comment (ashutosh97) [inline engines/dialogs.cpp]: Got added by mistake. Removed them too.

File: engines/dialogs.cpp (hunk 4 above)
Comment (ashutosh97) [inline engines/dialogs.cpp]: I removed the version name in the GMM and instead of that i added the game name. The reason behind doing so was that the version name is already displayed in the beginning when we launch ScummVM.exe , hence i thought that it would be useful if the game title is displayed when the user pauses the game by pressing Ctrl+F5. Displaying the game title would be useful in identifying the game which has been paused.

**Comment by hpvb** [comment]
Have we come to a decision on whether or not removing the ScummVM version from GMM header is a good idea or not? 

In the way it is done here adding the version also on the non-scaled gmm everything becomes super-cramped. If we want to keep the version and the game name in there I think a more thorough redesign of the gmm dialog is in order. 

Does anyone have any ideas on that?

**Comment by hpvb** [comment]
After discussion on IRC it seemed that:
 * Users are likely to know what game they are playing
 * Users are less likely to know what version of ScummVM they are on (particularly prepackaged ScummVM versions through retailers like GOG)
 * Most game names won't fit inside that space so it would look bad regardless

We're going to remove that task from the wiki.

Thanks for your submission though, please do keep contributing!

**Comment by sev-** [comment]
You could add the full name to a tooltip
